platform/web/os_web.h

```diff
@@ -88,9 +88,12 @@ class OS_Web : public OS_Unix {
 	String get_executable_path() const override;
 	Error shell_open(String p_uri) override;
 	String get_name() const override;
+
+#ifndef PROXY_TO_PTHREAD_ENABLED
```

Comment:
I'm not very fond of ifdefs in header files, can we move this to the CPP file instead?
```c++
void OS_Web::add_frame_delay(bool p_can_draw) {
#ifndef PROXY_TO_PTHREAD_ENABLED
        OS_Unix::add_frame_delay(p_can_draw);
#endif
}
```

Reply:
Done!